expounder.js

```diff
@@ -55,6 +55,10 @@
 
 					event.preventDefault();
 
+					// Only allow left and middle mouse clicks
+					if (event.button != 0 && event.button != 1)
```

Comment:
Hmm, what do we want to allow here? Which button numbers? Maybe we could just do `> 1`?